### PR TITLE
Swagger changes for a new feature which is to generate LOA for expressRoutePorts resource.

### PR DESCRIPTION
**Pull request by umkanodi**
…sRoutePorts resource

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by lirenhe — APPROVED** [review]

**Review by anton-evseev — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json [hunk 1]
```diff
@@ -542,6 +542,65 @@
...
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRoutePorts/{expressRoutePortName}/generateLoa": {
+      "post": {
+        "tags": [
+          "ExpressRoutePorts"
+        ],
+        "operationId": "ExpressRoutePorts_GenerateLOA",
+        "description": "Generate LOA for the requested ExpressRoutePort resource.",
```
Comment (majastrz) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json:552]: >LOA [](start = 33, length = 3)

Can you expand the LOA acronym in all the descriptions?

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json [hunk 2]
```diff
@@ -864,6 +923,27 @@
...
+        }
+      },
+      "required": [
+        "customerName"
+      ]
+    },
+    "GenerateExpressRoutePortsLOAResult": {
+      "description": "Response for GenerateExpressRoutePortsLOA API service call.",
+      "properties": {
+        "encodedPdf": {
```
Comment (majastrz) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json:942]: >encodedPdf [](start = 9, length = 10)

What is the maximum size of the PDF that you can potentially return here?

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json (hunk 1 above)
Comment (umkanodi) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json:552]: Letter of authorization

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json (hunk 2 above)
Comment (umkanodi) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json:942]: we are not returning the PDF file, rather converting the PDF file stream to Base64 encoded string, and returning the string.

**Comment by umkanodi** [comment]
I will make this PR against the may release branch. Closing this for now

**Pull request closed without merging**